_delphi_utils_python/delphi_utils/validator/report.py

```diff
@@ -90,6 +91,7 @@ def log(self, logger=None):
             logger.critical(str(error))
         for warning in self.raised_warnings:
             logger.warning(str(warning))
+        logger.info(self.summary)
```

Comment:
Move this up to before the errors and warnings are added to the logger so the summary is listed/displayed first.